base_export_async/models/delay_export.py

```diff
@@ -116,26 +119,29 @@ def export(self, params):
         expiration_date = fields.Date.to_string(
             date_today + relativedelta(days=+int(time_to_live)))
 
-        # TODO : move to email template
         odoo_bot = self.sudo().env.ref("base.partner_root")
         email_from = odoo_bot.email
         model_description = self.env[model_name]._description
-        self.env['mail.mail'].create({
-            'email_from': email_from,
-            'reply_to': email_from,
-            'recipient_ids': [(6, 0, users.mapped('partner_id').ids)],
-            'subject': _("Export {} {}").format(
-                model_description, fields.Date.to_string(fields.Date.today())),
-            'body_html': _("""
-                <p>Your export is available <a href="{}">here</a>.</p>
-                <p>It will be automatically deleted the {}.</p>
-                <p>&nbsp;</p>
-                <p><span style="color: #808080;">
-                This is an automated message please do not reply.
-                </span></p>
-                """).format(url, expiration_date),
-            'auto_delete': True,
-        })
+        mail_template = self.env.ref(
+            'base_export_async.delay_export_mail_template'
+        )
+        if mail_template:
```

Comment:
I always assume the template exists (or the module needs to be updated) in cases like this. But if you insist, you have to pass `raise_if_not_found=False` to `self.env.ref`.

Reply:
Done, raise_if_not_found=False added in self.env.ref